rca/project_styleguide/templates/patterns/pages/enquire_to_study/enquire_form_thanks.html

```diff
@@ -33,5 +33,15 @@ <h1 class="title-area__heading heading heading--display-two">
 
         </div>
     </div>
-
 {% endblock %}
+
+{% block extra_js %}
+    {% if GOOGLE_TAG_MANAGER_ID and enquiry_form_session_data %}
+        <script>
+            window.dataLayer = window.dataLayer || [];
+            window.dataLayer.push(
+                {{ enquiry_form_session_data|safe }}
```

Comment:
Applying the `safe` filter to form data could pose a security risk. If its causing JS errors it might be worth changing the format of the data.

Reply:
I've marked this as safe before the template https://github.com/torchbox/rca-wagtail-2019/pull/608/commits/d5668e341a0c8d00558148209992ab2b4a516d66